### PR TITLE
Update patch for v8

### PR DESCRIPTION
**Pull request by n30m1nd**
Update the patch so that it works with the latest version of v8.

### CODE REVIEW

File: Targets/V8/v8.patch [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
+index 915a781f3a..d6bc6689fc 100644
+--- a/src/api/api.cc
++++ b/src/api/api.cc
+@@ -503,7 +503,8 @@ void Utils::ReportOOMFailure(i::Isolate* isolate, const char* location,
+     if (fatal_callback == nullptr) {
+       base::OS::PrintError("\n#\n# Fatal %s OOM in %s\n#\n\n",
+                            is_heap_oom ? "javascript" : "process", location);
+-      base::OS::Abort();
++      exit(0);
++//      base::OS::Abort();
```
Comment (saelo) [inline Targets/V8/v8.patch:11]: Ah yeah, that's a good idea!

**Comment by saelo** [comment]
Thanks! Could you also update this file: https://github.com/googleprojectzero/fuzzilli/blob/master/Targets/V8/README.md to include the commit hash please?

**Comment by n30m1nd** [comment]
Done - it applies to the latest build according to `git apply --stat` so it should be good to go.

**Comment by saelo** [comment]
Thanks!

**Pull request merged**